tests/tests.py

```diff
@@ -57,9 +57,10 @@ def test_piranha_match_only():
         language="java",
         path_to_codebase="test-resources/java/structural_find/input",
         dry_run=True,
+        exclude=["*/folder_2_1/**/*"]
     )
     output_summaries = execute_piranha(args)
-    assert sum([len(summary.matches) for summary in output_summaries]) == 22
+    assert sum([len(summary.matches) for summary in output_summaries]) == 44
```

Comment:
It was accidental and I was lazy. But in theory i guess its alright. 
Instead of having a test of "match-only" rules and separate test for include/exclude. I thought I could combine the two. 
Mainly because, "structural-find-replace" tests a fundamental aspect of the program while include/exclude is just some configuration over it to determine which files to "include"/"exclude".

Reply:
Ok. I stopped being lazy, and added the test back. Now we have two tests :)